src/translations/en.json

```diff
@@ -971,6 +971,11 @@
             "name_error_msg": "Name is required",
             "linked_user": "Linked User",
             "device_tracker_intro": "Select the devices that belong to this person.",
+            "no_device_tracker_available_intro": "When you have devices that indicate the presence of a person, you will be able to assign them to a person here.",
+            "no_device_tracker_available_intro2": "You can add your first device by adding a",
+            "no_device_tracker_available_intro2a": "from the ",
+            "link_presence_detection_integration": "presence-detection integration",
+            "link_integrations_page": "integrations page",
```

Comment:
We shouldn't have half sentences in the translations, as they are impossible to translate. The order might be different in another language.

Reply:
I moved the links to a seperate list in the last commit, so we have a clean seperation for the localisation. It now looks like this:
![grafik](https://user-images.githubusercontent.com/1040402/66679791-61550e80-ec6f-11e9-8b6e-0856f4dd06f4.png)